docs/docs/flink.md

```diff
@@ -24,20 +24,20 @@ Apache Iceberg supports both [Apache Flink](https://flink.apache.org/)'s DataStr
 
 | Feature support                                             | Flink | Notes                                                                                  |
 | ----------------------------------------------------------- |-------|----------------------------------------------------------------------------------------|
-| [SQL create catalog](#creating-catalogs-and-using-catalogs) | ✔️    |                                                                                        |
-| [SQL create database](#create-database)                     | ✔️    |                                                                                        |
-| [SQL create table](#create-table)                           | ✔️    |                                                                                        |
-| [SQL create table like](#create-table-like)                 | ✔️    |                                                                                        |
-| [SQL alter table](#alter-table)                             | ✔️    | Only support altering table properties, column and partition changes are not supported |
-| [SQL drop_table](#drop-table)                               | ✔️    |                                                                                        |
-| [SQL select](#querying-with-sql)                            | ✔️    | Support both streaming and batch mode                                                  |
-| [SQL insert into](#insert-into)                             | ✔️ ️  | Support both streaming and batch mode                                                  |
-| [SQL insert overwrite](#insert-overwrite)                   | ✔️ ️  |                                                                                        |
-| [DataStream read](#reading-with-datastream)                 | ✔️ ️  |                                                                                        |
-| [DataStream append](#appending-data)                        | ✔️ ️  |                                                                                        |
-| [DataStream overwrite](#overwrite-data)                     | ✔️ ️  |                                                                                        |
-| [Metadata tables](#inspecting-tables)                       | ✔️    |                                                                                        |
-| [Rewrite files action](#rewrite-files-action)               | ✔️ ️  |                                                                                        |
+| [SQL create catalog](../flink-ddl.md#create-catalog) | ✔️    |                                                                                        |
```

Comment:
it seems only this row was updated with the whitespacing changes but not the others

Reply:
are you referring to a previous change? In this PR, this line has no whitespace change

Reply:
nvm. Formatting these tables is quite annoying whenever some text changes, so let's just go with the diff you have